source/camera/basic/capture.py

```diff
@@ -7,13 +7,13 @@ def _main():
     app = zivid.Application()
     camera = app.connect_camera()
 
-    with camera.update_settings() as updater:
-        updater.settings.iris = 20
-        updater.settings.exposure_time = datetime.timedelta(microseconds=6500)
-        updater.settings.filters.outlier.enabled(True)
-        updater.settings.filters.outlier.threshold(5)
+    settings = zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(aperture=5.66, exposure_time=datetime.timedelta(microseconds=8333),),
+        ],
+    )
```

Comment:
You could also consider this if you want:
```
settings = zivid.Settings()
settings.acquisitions.append(zivid.Settings.Acquisition())
settings.acquisitions[0].aperture = 5.66
settings.acquisitions[0].exposure_time = datetime.timedelta(microseconds=8333)
```

Reply:
Done